pages/docs/concepts/asyncapi-document/resuable-parts.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Reusable parts
+weight: 249
+---
+
+Reusable parts in AsyncAPI provide flexibility, modularity, and code reusability. Using reusable parts makes it easier to generate code and validate the specifications. You can reuse specific document sections such as Messages or schema definitions.
+
+Reusable parts allow you to split up the AsyncAPI document into many files and reference them using the Reference Object ($ref). You can use the $ref keyword to reference the same or another local file or external URL.
+
+## Same file
+
+You can use the ref keyword to reference a component within the same document. The image below defines referencing a component within the same document.
```

Comment:
I notice you're using the word `image` to introduce the diagrams in this PR, but we should be consistent across the documentation. Currently, we are using the word `diagram`. 

Please update all your mentions accordingly. 😄